action.yml

```diff
@@ -9,6 +9,12 @@ inputs:
     description: GitHub token
     required: false
     default: ${{github.token}}
+  owner:
+    description: Owner of the repository. Will be deducted from github_token if it is not provided
+    required: false
+  repository:
+    description: The repository containing the branch(es) to be deleted. Will be deducted from github_token if it is not provided
```

Comment:
Repository name will be deducted from env vars, not from github_token

Reply:
Let's talk about this when we both have time

Reply:
Description changed for both 'owner' and 'repository'